frontend/src/mock-cloud/FakeCloudDashboard.tsx

```
@@ -0,0 +1,33 @@
+    return null;
+  }
+
+  return (
+    <div className="flex h-screen flex-col">
+      <FakeCloudNavbar />
+      <div className="min-h-0 flex-1">
+        <InsForgeDashboard
+          mode="cloud-hosting"
+          project={FAKE_PROJECT}
+          {...STUB_CALLBACKS}
+        />
```

ESLint

[error] Replace `⏎··········mode="cloud-hosting"⏎··········project={FAKE_PROJECT}⏎··········{...STUB_CALLBACKS}⏎·······` with `·mode="cloud-hosting"·project={FAKE_PROJECT}·{...STUB_CALLBACKS}`

(prettier/prettier)